tests/full.bats

```diff
@@ -0,0 +1,55 @@
+load helpers/bats-support/load.bash
+load helpers/bats-assert/load.bash
+
+setup_file() {
+  if [ -n "$TEST_DRUPAL_CORE" ] && [ "$TEST_DRUPAL_CORE" != "10" ] && [ "$TEST_DRUPAL_CORE" != "11"  ]; then
+    skip "TEST_DRUPAL_CORE=$TEST_DRUPAL_CORE not handled by this test suite" >&2
+  fi
+  load '_common.bash'
+  _common_setup
+  ddev config --php-version=8.2
+  if [ "$TEST_DRUPAL_CORE" = "11" ]; then
+    ddev config --php-version=8.3 --corepack-enable
+  fi
```

Comment:
I recommend just `ddev config --auto`, which does all these things.

```suggestion
  ddev config --auto
```

Reply:
How does ddev config --auto does that? Basically I wanted to test d10 without corepack and d11 with, and also with both different php versions, not really needed, but figure it wasn't too bad

Reply:
🤔 I think you implied that it does that be detecting d11, but d11 would not be installed yet on this case as it's taken care by `ddev poser` so I think this would not be the case.

Reply:
I would leave it like this for now if that's ok, it is also more verbose.